src/style.css

```diff
@@ -0,0 +1,168 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+header {
+  padding: 0 50px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+}
+
+li {
+  margin-left: 20px;
+  display: flex;
+}
+
+li:first-child {
+  margin: 0;
+}
+
+a {
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 23px 0;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  background: #00acdc;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  content: "";
+}
+
+/*here is product card*/
+
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 400px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3f3;
+}
+
+.image {
+  padding: 31px 20px 1px;
+  height: 132px;
+  width: 158px;
+  border-top: 1px;
+}
+
+.title {
+  padding: 45px 15px 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.item_code {
+  padding: 5px 15px 0;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.rank {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 14px 15px 0;
+
+}
+
+.stars {
+  display: flex;
+  padding-left: 1px;
+
+}
+
+.star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+}
+
+.reviews {
+  padding-top: 2px;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.price {
+  padding: 21px 15px 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.price_title {
+  font-size: 11px;
+  line-height: 13px;
+  padding-top: 5px;
+}
+
+.price_number {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.buy {
+  margin: 20px 15px 0;
+}
+
+.button_text {
+  border: 1px solid #00acdc;
+  font-weight: bold;
+  line-height: 16px;
+  font-size: 14px;
+  text-transform: uppercase;
+  background: #00acdc;
+  border-radius: 5px;
+  text-align: center;
+  text-decoration: none;
+  color: #fff;
+  display: block;
+  padding: 12px;
+}
+
+.button_text:hover {
+  background-color: #fff;
+  color: #00acdc;
+}
+
+main {
+  margin: 50px 40px 40px;
+}
+
+.main {
```

Comment:
Bad name for a class.
In your case, i've recommended something like `container`.